examples/ndcube_from_radio_images_example.py

```diff
@@ -0,0 +1,152 @@
+"""
+=================
+Creating an NDCube from solar radio interferometric images.
+=================
+
+How to make an NDCube from solar radio interferometric images.
+
+The example uses `dkist` https://docs.dkist.nso.edu/projects/python-tools/en/latest/ and `gwcs`
+to create a single wcs object that takes into account the changing crval of 
+solar radio images using `~dkist.wcs.models.VaryingCelestialTransform`. 
+The following example uses data from the NenuFAR radio telescope
+https://nenufar.obs-nancay.fr/en/homepage-en/
+"""
+import glob
+
+import numpy as np
+
+from multiprocessing import Pool
+
+from astropy.modeling import models
+from astropy import coordinates as coord
+from astropy.coordinates import SkyCoord, EarthLocation
+from astropy import units as u
+from astropy.io import fits
+from astropy.time import Time, TimeDelta
+from gwcs import wcs
+from gwcs import coordinate_frames as cf
+from ndcube import NDCube
+from sunpy.coordinates.frames import Helioprojective
+from sunpy.coordinates import sun
+
+from dkist.wcs.models import VaryingCelestialTransform
+
+##############################################################################
+# We first need to load some data. These exist on a machine somewhere and 
+# I don't know how to share them. 
+# Here we take every image at a single frequency
+
+ims = glob.glob("/data/mpearse/radio_images/SB158/*image.fits")
+ims.sort()
+
+def get_data(file):
```

Comment:
Would this example be just as helpful if you created fake data in this function, e.g. using `numpy.random` or `numpy.ones`, and explicitly defined the time, frequency, etc., instead of reading it from a file?  That way you wouldn't have to worry about how to share the files.

Reply:
:+1: definitely, yeah! That's much easier.